src/main/java/net/library/util/Utils.java

```diff
@@ -11,4 +16,19 @@ public static String getUUID() {
     public static String deleteSpacesHyphens(String value) {
         return value.replace(" ", "").replace("-", "").replace("+", "");
     }
-}
+
+    public static LocalDateTime stringToLocalDateConverter(String value) {
+        if (value == null || value.isEmpty() || value.isBlank()) {
+            return null;
+        }
+        var dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm");
```

Comment:
Static method 'Utils. localDateConverter()' does the same thing.
Replace or remove extra code